core/src/test/scala/org/apache/spark/status/AppStatusStoreSuite.scala

```diff
@@ -133,9 +154,54 @@ class AppStatusStoreSuite extends SparkFunSuite {
 
   private def newTaskData(i: Int, status: String = "SUCCESS"): TaskDataWrapper = {
     new TaskDataWrapper(
-      i, i, i, i, i, i, i.toString, i.toString, status, i.toString, false, Nil, None,
+      i.toLong, i, i, i, i, i, i.toString, i.toString, status, i.toString, false, Nil, None, true,
       i, i, i, i, i, i, i, i, i, i,
       i, i, i, i, i, i, i, i, i, i,
       i, i, i, i, stageId, attemptId)
   }
+
+  private def writeTaskDataToStore(i: Int, store: KVStore, status: String): Unit = {
+   val liveTask = new LiveTask(new TaskInfo( i.toLong, i, i, i.toLong, i.toString,
```

Comment:
indentation is off

Reply:
Right, not sure why style check didn't catch this.